test/test_git_helpers.py

```diff
@@ -0,0 +1,43 @@
+import time
+
+import conda_build_prepare.git_helpers as gh
+
+COMMIT_DELAY = 1
+
+
+def git_commit_and_tag(filepath, msg, tag=None):
+    now = time.time()
```

Comment:
Why not use `tempfile`?

Reply:
I do use `tmp_path` fixture from pytest in the test function, see https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html for more details.

The date here is used for the file content (not the name).